pymc3/tuning/starting.py

```diff
@@ -30,21 +34,21 @@ def find_MAP(start=None, vars=None, fmin=None,
     start : `dict` of parameter values (Defaults to `model.test_point`)
     vars : list
         List of variables to set to MAP point (Defaults to all continuous).
-    fmin : function
-        Optimization algorithm (Defaults to `scipy.optimize.fmin_bfgs` unless
```

Comment:
Do we need to deprecate `fmin` rather than getting rid of it completely right away? This might cause a fair amount of breakage. Thoughts?